src/test/java/teammates/test/cases/util/SanitizationHelperTest.java

```diff
@@ -358,4 +358,24 @@ public void testDesanitizeIfSanitized() {
         assertEquals("should return same unsanitized string if given unsanitized string containing sanitized sequences",
                 unsanitized, SanitizationHelper.desanitizeIfHtmlSanitized(unsanitized));
     }
+
+    @Test
+    public void testSanitizeForLogMessage() {
+        assertNull("should return null if given null", SanitizationHelper.sanitizeForLogMessage(null));
+
+        String unsanitized = "<span class=\"text-danger\"> A <span class=\"bold\">typical</span> log  message <br>"
+                + " It contains some <script>dangerous</script> elements </span>";
+        String correctSanitized =
+                "<span class=\"text-danger\"> A <span class=\"bold\">typical</span> log  message <br>"
+                + " It contains some &lt;script&gt;dangerous&lt;&#x2f;script&gt; elements </span>";
+        assertEquals("Should escape HTML special characters"
+                + "other than in <span class=\"text-danger\">, <span class=\"bold\"> and <br>",
+                correctSanitized, SanitizationHelper.sanitizeForLogMessage(unsanitized));
+
+        unsanitized = "Hmm. <span class=\"text-info\"> How about this? </span> and <span> this</span>";
+        correctSanitized =
+                "Hmm. &lt;span class=&quot;text-info&quot;&gt; How about this? </span> and <span> this</span>";
+        assertEquals("Should escape <span class=\"text-danger\">, <span class=\"bold\"> and <br>",
```

Comment:
The message is bit confusing
`Should escape tag other than <span class=\"text-danger\">, <span class=\"bold\"> and <br>` right?

Reply:
My mistake. Copy pasted from above and didn't delete all